mzLib/Readers/ExternalResults/ResultFiles/MsPathFinderTResultFile.cs

```diff
@@ -10,18 +10,12 @@ namespace Readers
 {
     public class MsPathFinderTResultFile : ResultFile<MsPathFinderTResult>, IResultFile
     {
-        public override SupportedFileType FileType { get; }
+        public override SupportedFileType FileType => FilePath.ParseFileType();
```

Comment:
Problematic. In any solution which uses mzlib, we could construct a ResutlFile<T> from any record whatsoever. This would not have a supported file type, but retain all other result file operations. 

With this change, we would get an exception exception thrown frequently